lib/prometheus_exporter/server/sidekiq_stats_collector.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module PrometheusExporter::Server
+  class SidekiqStatsCollector < TypeCollector
+    MAX_SIDEKIQ_METRIC_AGE = 60
+
+    SIDEKIQ_STATS_GAUGES = {
+      'dead_size' => 'Size of dead the queue',
+      'enqueued' => 'Number of enqueued jobs',
+      'failed' => 'Number of failed jobs',
+      'processed' => 'Total number of processed jobs',
+      'processes_size' => 'Number of processes',
+      'retry_size' => 'Size of the retries queue',
+      'scheduled_size' => 'Size of the scheduled queue',
+      'workers_size' => 'Number of jobs being worked',
```

Comment:
hmm this description seems a bit off. Perhaps:

"Number of jobs actively being processed"

Reply:
👍🏻 https://github.com/discourse/prometheus_exporter/pull/194/commits/dee7768fac9fdd31782f45ef9ba53aee6251fcd6